projects/tuist/fixtures/ios_app_with_custom_regions/Project.swift

```diff
@@ -0,0 +1,19 @@
+import ProjectDescription
+
+let project = Project(
+    name: "App",
+    options: .options(
+        defaultKnownRegions: ["en-GB", "Base"],
+        developmentRegion: "en-GB"
+    ),
+    targets: [
+        Target(
+            name: "App",
+            platform: .iOS,
+            product: .app,
+            bundleId: "io.tuist.app",
+            infoPlist: "App/Info.plist",
```

Comment:
minor, we can lean on the auto generated info plist file

```suggestion
            infoPlist: .default,
```